libraries/datahandler/src/main/thrift/components.thrift

```diff
@@ -405,6 +405,7 @@ struct ComponentDTO {
     51: optional string mailinglist,
     52: optional string wiki,
     53: optional string blog,
+    56: optional string vcs,
```

Comment:
Please add the appropriate json ignore for this new field (like `setVcs`)

https://github.com/eclipse-sw360/sw360/blob/dfc187d9f609fa6644b8d5e38752c31236288d86/rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java#L826